### DISCUSSION TITLE
关于下载？

### DISCUSSION THREAD

**Discussion by hrwxl** [banned-historical-archives/banned-historical-archives.github.io#4035] [General]
请问仓库文件如何批量下载？

**Comment by banned-historical-archives** [comment]
需要学习一下git命令行，如果不懂我晚点做一个辅助工具

**Reply by hrwxl** [reply]
这里面是我手里的一些资料，有些应该是没有的https://almaelm-my.sharepoint.com/:f:/g/personal/zzz_almaelm_onmicrosoft_com/Ev7r5hiXdihEgVXg9vR4LpwBHM5z065n9LsKEM23bCGzIg?e=XfGJU8

**Reply by banned-historical-archives** [reply]
> 这里面是我手里的一些资料，有些应该是没有的https://almaelm-my.sharepoint.com/:f:/g/personal/zzz_almaelm_onmicrosoft_com/Ev7r5hiXdihEgVXg9vR4LpwBHM5z065n9LsKEM23bCGzIg?e=XfGJU8

感谢，我看看

**Reply by hrwxl** [reply]
顺便有个问题想问下您，有些报刊的光盘和数据比如解放军报，我在国内也没有看到有获取的渠道，想了解是什么渠道得到的

**Reply by banned-historical-archives** [reply]
> 顺便有个问题想问下您，有些报刊的光盘和数据比如解放军报，我在国内也没有看到有获取的渠道，想了解是什么渠道得到的

别人给我的，源头不清楚
